### ISSUE TITLE
FeatureRequest / Query: Config for Numerical Metric Based Parameters

### ISSUE THREAD

**Issue by luke-hill** [bbatsov/rubocop#5902]
## Expected behavior

For Numerical Metric based cops (Or infact, all cops to an extent), I want to be able to configure these values to be different in different "areas" of the suite.

## Actual behavior

I can enable and disable cops, but I cannot "configure / tweak" cops across the suite. (To my knowledge) - Apologies if you can

## Steps to reproduce the problem

In a cucumber project. You will regularly break the 80character limit on your gherkin files. These aren't configured to be monitored by rubocop :+1: 
However, these then also have a corresponding step definition, these will be monitored by rubocop.

I want to have something like this...

```
AllCops:
  LineLength:
    Matcher: './**/step_definitions/*_steps.rb' # New Logic
      Max: 120
    Otherwise:
      Max: 80
```

Apologies if this is already configurable. If not I'd like to hear some opinions from contributors as to.

a) Is this possible?
b) Is this desirable (My opinion is yes) - If originally introduced, maybe just on numerical metric based ones

**Comment by mikegee** [comment]
Try making `step_definitions/.rubocop.yml` with:

```yaml
Metrics/LineLength:
  Max: 120
```

**Comment by luke-hill** [comment]
For 15 intermediate level folders (If this works), would require me creating this file 15 times though?

I have the following structure
`features-> Around 20 folders (Most of which have a steps folder) -> between 1 and 6 features
